src/themes/admin_future/modules/contact/view.tpl

```diff
@@ -339,20 +181,149 @@
+    <div class="modal-dialog modal-lg" role="document">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h4 class="modal-title">{$LANG->getModule('send_title')}</h4>
+                <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+            </div>
+            <div class="modal-body">
+                <form method="post" class="modal-body">
+                    <div class="card">
+                        <div class="card-body">
+                            <div class="row mb-3">
+                                <div class="col-sm-2 col-form-label text-sm-end">{$LANG->getModule('title_send_title')}</div>
+                                <div class="col-sm-9">
+                                    <input name="title" type="text" value="Re:{$DATA.title}" class="form-control" disabled="true">
+                                </div>
+                            </div>
+                            <div class="row mb-3">
+                                <div class="col-sm-2 col-form-label text-sm-end">{$LANG->getModule('email')}</div>
+                                <div class="col-sm-9">
+                                    <input name="email" type="email" value="{$DATA.sender_email}" class="form-control" disabled="true">
+                                </div>
+                            </div>
+                            <div class="row">
+                                {$MESS_CONTENT}
+                            </div>
+                        </div>
+                        <div class="card-footer text-center">
+                            <button type="submit" class="btn btn-primary">{$LANG->getModule('bt_send_row_title')}</button>
+                            <button type="button" class="btn btn-secondary" data-bs-dismiss="modal">{$LANG->getGlobal('close')}</button>
+                        </div>
+                    </div>
+                    <input type="hidden" name="reply" value="{$DATA.id}">
+                    <input type="hidden" name="checkss" value="{$CHECKSS}">
+                </form>
+            </div>
+        </div>
+    </div>
+</div>
+
+<div class="modal fade" id="feedback-forward" data-backdrop="static" tabindex="-1" role="dialog">
+    <div class="modal-dialog modal-lg" role="document">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h4 class="modal-title">{$LANG->getModule('mark_as_forward')}</h4>
+                <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+            </div>
+            <div class="modal-body">
+                <form method="post" class="modal-body">
+                    <div class="card">
+                        <div class="card-body">
+                            <div class="row mb-3">
+                                <div class="col-sm-2 col-form-label text-sm-end">{$LANG->getModule('title_send_title')}</div>
```

Comment:
@danlt150 cột này em check lại dùng `col-sm-2` có vẻ quá nhỏ, xl-2 thì ok, còn dưới nó nên để to ra cỡ 3 - 4

Reply:
Tương tự cho modal bên trên nó là cái reply